src/components/ContactInformation/ContactInformationContent.ts

```diff
@@ -0,0 +1,29 @@
+import { Picture } from '../../model/Picture';
+
+/**
+ * @title Карточка с контактной информацией
+ * @default {
+ *   "format": "webp",
+ *   "size": {
+ *       "width": 48
+ *   }
+ */
+export interface ContactCard {
+  icon?: Picture;
+  label?: string;
+  data?: ContactData[];
+}
+/**
+ * @title Контактная информация
+ */
+export interface ContactData {
+  label?: string;
+  values?: string[];
+}
+/**
+ * @title Список картчек с контактной информации
+ */
+export interface ContactInformationContent {
+  title?: string;
```

Comment:
И так надо сделать везде, где `@title` не задан (исключение, если тип не примитивный, а ссылается например на `Picture`)

```suggestion
  /** @title Заголовок */
  title?: string;
```